docs/reference/commands/node-tool.asciidoc

```diff
@@ -2,14 +2,15 @@
 == elasticsearch-node
 
 The `elasticsearch-node` command enables you to perform unsafe operations that
-risk data loss but which may help to recover some data in a disaster.
+risk data loss but which may help to recover some data in a disaster and
+repurpose nodes into new roles.
 
 [float]
 === Synopsis
 
 [source,shell]
 --------------------------------------------------
-bin/elasticsearch-node unsafe-bootstrap|detach-cluster
+bin/elasticsearch-node unsafe-bootstrap|detach-cluster|repurpose
```

Comment:
I'm inclined to put the `repurpose` command first here (and elsewhere) because of the three commands it's the one that we hope is the most used.